src/tools/diagnostic-messages.md

```diff
@@ -1992,6 +2207,42 @@ suitable value:
 var l = const [0];
 {% endprettify %}
 
+### compound_implements_finalizable
+
+_The class '{0}' can't implement Finalizable._
+
+#### Description
+
+The analyzer produces this diagnostic when a subclass of either `Struct`
+or `Union` implements `Finalizable`.
+
+For more information about FFI, see [C interop using dart:ffi][].
```

Comment:
@bwilkerson It seems you never included the link target for these anywhere in the generated Markdown output.